src/patternfly/components/Button/styles.scss

```diff
@@ -1,6 +1,6 @@
 @import "../../patternfly-utilities";
 
-:root {
+.pf-c-button {
```

Comment:
If I am reading this right won't this make it harder to theme? Instead of being able to do something like:
```css
.some-cutom-theme {
  --pf-c-button--m-primary--focus--BackgroundColor: #000;
}
```
I now need to do 
```css
.some-custom-theme .pf-c-button {
  --pf-c-button--m-primary--focus--BackgroundColor: #000;
}
```
This requires me as a consumer to know both the variable name and the class name.  Since the variables are already scoped with the `pf-{type}-{component}--{m}-{state}--{PropertyName}` I would be in favor of leaving these at `:root` so that I don't need to know the classname as well.

Reply:
+1 What @dmiller9911 is saying makes sense.

Reply:
If we leave it on root the value variable doesn't change. It's something we've learned from custom properties while working on this.

If you want lets talk and I can show you why

Reply:
@andresgalante thanks for the demo on why you must do it the way you did.